x-pack/plugins/uptime/public/pages/monitor.tsx

```diff
@@ -55,12 +56,23 @@ export class MonitorPage extends React.Component<MonitorPageProps> {
     return (
       <Fragment>
         <EuiTitle>
-          <h2>{id}</h2>
+          <h2>
+            <FormattedMessage
```

Comment:
There is nothing to translate. It's just a variable `id`.

Reply:
Yikes - we'll say this was one of the first ones I added 👼

Reply:
Resolved in https://github.com/elastic/kibana/pull/28183/commits/8a4355099a85704847a0effeaff0aec9f7906413.